indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -127,28 +127,31 @@ def _process(self):
 class RHManageParticipants(RHManageRegFormsBase):
     """Show and enable the dummy registration form for participants."""
 
-    def _process_POST(self):
-        regform = self.event.participation_regform
-        set_feature_enabled(self.event, 'registration', True)
-        if not regform:
-            regform = RegistrationForm(event=self.event, title='Participants', is_participation=True,
-                                       currency=payment_settings.get('currency'),
-                                       publish_registrations_public=(PublishRegistrationsMode.hide_all
-                                                                     if self.event.type_ == EventType.conference
-                                                                     else PublishRegistrationsMode.show_with_consent))
-            create_personal_data_fields(regform)
-            db.session.add(regform)
-            db.session.flush()
-            signals.event.registration_form_created.send(regform)
-            self.event.log(EventLogRealm.management, LogKind.positive, 'Registration',
-                           f'Registration form "{regform.title}" has been created', session.user)
-        return redirect(url_for('event_registration.manage_regform', regform))
-
-    def _process_GET(self):
+    def _process(self):
         regform = self.event.participation_regform
         registration_enabled = self.event.has_feature('registration')
+        form = RegistrationFormCreateForm(event=self.event, title='Participants',
+                                          currency=payment_settings.get('currency'),
+                                          visibility=[PublishRegistrationsMode.show_with_consent.name,
+                                                      PublishRegistrationsMode.hide_all.name])
```

Comment:
I think here we may want different behavior for meetings and other event types... for lectures/conferences I think the default is fine, but for meetings I'd assume a "show to everyone" default - or if we can't do that, force an active choice (because most meetings do NOT use a registration, just a participant list, so there's no real opportunity where someone would be able to give that consent)

Reply:
That's a good point, I updated the defaults to be "show to everyone" for meetings and "hide to everyone for conferences"